sw/airborne/modules/relativelocalizationfilter/arrayfunctions.c

```diff
@@ -0,0 +1,294 @@
+/*
+ * Copyright (C) Mario Coppola
+ *
+ * This file is part of paparazzi
+ *
+ * paparazzi is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * paparazzi is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with paparazzi; see the file COPYING.  If not, see
+ * <http://www.gnu.org/licenses/>.
+ */
+/**
+ * @file "modules/relativelocalizationfilter/arrayfunctions.c"
+ * @author Mario Coppola
+ * Array functions for array processing
+ */
+
+#include "arrayfunctions.h"
```

Comment:
we usually include paths from sw/airborne. So here it should be `#include "modules/relativelocalizationfilter/arrayfunctions.h"`.
As stated by Kirk, you should also add `_` between words to improve readability.